Frontend/ui-common/src/Engineering/Helpers/Commons.js

```diff
@@ -459,3 +459,26 @@ export const getImageUrl = function (url) {
     console.log("error in getImageUrl " + e);
   }
 };
+
+export const getTimestampObject = function(string){
+  window.jsonObject[string] = Date.now() - window.prevTimeStamp;
+  window.prevTimeStamp = Date.now();
+  if (string == "baseAppFlow")
+    window.tobeLog = true;
+  else
+    window.tobeLog = false;
+}
```

Comment:
add check to skip if key name is already present

Reply:
Why do we need this? Even if the user comes back to the same flow the `tobeLog` check will handle this right as it will not come inside this function only.

Reply:
we should always deal with any edge cases in present that may cause issue in future, because if some code refactor happens then it might happen that it will not log accurate result